tpm2/tpm2.go

```diff
@@ -87,7 +87,12 @@ func decodeTPMLPCRSelection(buf *bytes.Buffer) (PCRSelection, error) {
 	if err := tpmutil.UnpackBuf(buf, &count); err != nil {
 		return sel, err
 	}
-	if count != 1 {
+	switch count {
+	case 0:
+		sel.Hash = AlgUnknown
+		return sel, nil
+	case 1:
```

Comment:
nit: For clarity, please annotate the `case 1` statement with a comment to say why it's a no-op and `case 0` isn't.